pkg/kubectl/cmd/run.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2014 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl"
+	"github.com/spf13/cobra"
+)
+
+func (f *Factory) NewCmdRunContainer(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "run-container <name> --image=<image> [--command=<some command here>] [--replicas=replicas] [--dry-run=<bool>]",
+		Short: "Run a particular image on the cluster.",
+		Long: `Create and run a particular image, possibly replicated.
+Creates a replication controller to manage the created image(s)
```

Comment:
"the created containers(s)", to avoid confusion if/when/where there is a way to create images (e.g., in Openshift)